### ISSUE TITLE
Error with the first import on Windows

### ISSUE THREAD

**Issue by xBorja042** [SukiCZ/cardmarket_api#1]
* Cardmakret API v3+ version: 0.1.0
* Python version: 3.9.12
* Operating System: Windows 10

### Description

I was just toying around with the library and right after the installation I tried to make the first import and I got an error.

```
from cardmarket_api import CardMarketApi

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda3\lib\site-packages\cardmarket_api\__init__.py", line 8, in <module>
    from .cardmarket_api import CardMarketApi
  File "C:\Anaconda3\lib\site-packages\cardmarket_api\cardmarket_api.py", line 4, in <module>
    from . import clients
  File "C:\Anaconda3\lib\site-packages\cardmarket_api\clients\__init__.py", line 1, in <module>
    from .global_client import GlobalClient
  File "C:\Anaconda3\lib\site-packages\cardmarket_api\clients\global_client.py", line 1, in <module>
    from .base import BaseClient
  File "C:\Anaconda3\lib\site-packages\cardmarket_api\clients\base.py", line 3, in <module>
    from clients.exceptions import ApiException
ModuleNotFoundError: No module named 'clients'
```
I hope you can fix it and this helps. It is my first opened issue, I hope it is well posted

**Comment by SukiCZ** [comment]
> hopefully soon

That was sooner than I expected 😄 - Fixed in version `0.1.1`+

I'd also updated Usage in REDME / [docs](https://cardmarket-api.readthedocs.io/en/latest/usage.html) with information how to list a product on MarketPlace - hope this will help.

Any further feedback will be appreciated.

**Comment by xBorja042** [comment]
Yes, now at least I pass through that import. Maybe today I will have some time to test the API and give you feedback and maybe in the future I could contribute to the library with some code or an example notebook if applicable. Great job!
